### PR TITLE
semicolon on comment issue solved. Issues #17635

### PR DESCRIPTION
**Pull request by hameddanesh**
Semicolon repeatedly suffixed onto SQL if query ends in comment #17635 fixed.

### CODE REVIEW

File: po/az.po [hunk 1]
```diff
@@ -24,7 +24,7 @@ msgstr "PDF yaradılarkən xəta baş verdi:"
 #: libraries/classes/Advisory/Advisor.php:267
 #, php-format
 msgid "Failed evaluating precondition for rule '%s'."
-msgstr ""
+msgstr "%s qanunı üçün ilkin şərt qiymətdirilmədi"
```
Comment (williamdes) [inline po/az.po:27]: Thank you for translating phpMyAdmin
But please remove the changes from this PR and post them on Weblate: https://hosted.weblate.org/projects/phpmyadmin/5-2/az/

File: libraries/classes/Controllers/Import/ImportController.php [hunk 2]
```diff
@@ -131,7 +131,7 @@ public function __invoke(ServerRequest $request): void
 
         // If it's a console bookmark add request
         if (isset($_POST['console_bookmark_add'])) {
-            if (! isset($_POST['label'], $_POST['db'], $_POST['bookmark_query'], $_POST['shared'])) {
+            if (!isset($_POST['label'], $_POST['db'], $_POST['bookmark_query'], $_POST['shared'])) {
```
Comment (williamdes) [inline libraries/classes/Controllers/Import/ImportController.php:134]: Please remove all the non necessary changes made on the code, its probably your IDE that did the automatically

Comment (hameddanesh) [reply libraries/classes/Controllers/Import/ImportController.php:134]: yes it does auto-format
i'll fix it right now

File: po/az.po (hunk 1 above)
Comment (hameddanesh) [reply po/az.po:27]: sure, i'll get right on it

**Pull request closed without merging**